### PR TITLE
Gol glider

### PR DESCRIPTION
**Pull request by riveSunder**
changes to make loading of ca_configs less broken

### CODE REVIEW

File: coverage.txt [hunk 1]
```diff
@@ -15,9 +15,9 @@ yuca/params_agent.py                  32     11    66%
...
 yuca/wrappers/__init__.py              3      0   100%
-yuca/wrappers/glider_wrapper.py       88     12    86%
-yuca/wrappers/halting_wrapper.py     182     28    85%
-yuca/wrappers/random_wrapper.py       16      4    75%
+yuca/wrappers/glider_wrapper.py       90     13    86%
+yuca/wrappers/halting_wrapper.py     184     29    84%
+yuca/wrappers/random_wrapper.py       18      5    72%
 ------------------------------------------------------
-TOTAL                               2016    279    86%
+TOTAL                               2034    283    86%
```
Comment (riveSunder) [inline coverage.txt:22]: ✓

File: yuca/ca/neural.py [hunk 2]
```diff
@@ -299,7 +310,8 @@ def get_params(self):
         for hh, param in enumerate(self.weights_layer.named_parameters()):
             params = np.append(params, param[1].detach().cpu().numpy().ravel())
 
-        params = np.append(params, self.kernel_params)
+        if self.kernel_params is not None:
```
Comment (riveSunder) [inline yuca/ca/neural.py:313]: a better fix in future will be to stop having multiple copies of things (e.g. `kernel_params` as a class variable and those same params in `neighborhood_kernel_config` in the `ca_config`

* TODO

File: yuca/wrappers/glider_wrapper.py [hunk 3]
```diff
@@ -59,6 +59,10 @@ def sample(self):
 
             return torch.rand(*self.shape, dtype=torch.get_default_dtype())
 
+    def eval(self):
+
+        self.ca.eval()
```
Comment (riveSunder) [inline yuca/wrappers/glider_wrapper.py:64]: For turning dropout off during e.g. glider evolution.
